packages/libs/react-ui/src/components/Button/NewButton.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable @kadena-dev/no-eslint-disable */
+
+import { useObjectRef } from '@react-aria/utils';
+import type { RecipeVariants } from '@vanilla-extract/recipes';
+import type {
+  ComponentProps,
+  ForwardedRef,
+  MouseEvent,
+  ReactNode,
+} from 'react';
+import React, { forwardRef, useCallback } from 'react';
+import type { AriaButtonProps } from 'react-aria';
+import { useButton } from 'react-aria';
+import { Loading } from '../Icon/System/SystemIcon';
+import { button, spinner } from './NewButton.css';
+
+type Variants = Omit<NonNullable<RecipeVariants<typeof button>>, 'onlyIcon'>;
+export interface IButtonProps extends AriaButtonProps, Variants {
+  startIcon?: ReactNode;
+  endIcon?: ReactNode;
+  icon?: ReactNode;
+  title?: string;
```

Comment:
can we remove this? I believe people can use `aria-label` for this

Reply:
this is a different thing, this is the native `title` prop which is used to add a native tooltip and it is useful for icon buttons